src/EditorFeatures/CSharpTest/Completion/CompletionProviders/OverrideCompletionProviderTests.cs

```diff
@@ -3062,5 +3065,20 @@ public async Task OverrideInRecordWithExplicitOverriddenMember()
     override $$
 }", "ToString()");
         }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(47973, "https://github.com/dotnet/roslyn/issues/47973")]
+        public async Task NoCloneInOverriddenRecord()
+        {
+            var cloneMemberName = (string)typeof(WellKnownMemberNames).GetField("CloneMethodName", BindingFlags.NonPublic | BindingFlags.Static).GetValue(null);
+            Assert.Equal("<Clone>$", cloneMemberName);
```

Comment:
intentional.  this isn't documented by teh compiler, and i don't want to hard code in the name in a 'missing' check.  i.e. fi the compiler change the name to something else and that now shows up, this test woudl still pass, but we might have bad behavior.  so i'm actually intentionally taking a dependency on this, and i want this to fire if they cahnge things.

Reply:
Should the test that the value shouldn't change just live in the compiler?

If nothing, I wonder if this deserves a good comment in the code. If I saw this I'd go "WTF" and just delete it without a second thought.

Reply:
> Should the test that the value shouldn't change just live in the compiler?

I don't see how that helps us.  compiler changes the name, updates tehir test, IDE may silently start breaking.  i will doc this code though.

Reply:
I guess the behavior you're trying to assert is that if the generated name changes, any code using the existing unspeakable name doesn't break, right? Does the compiler have some sort of test binary we can reference in their resources that contains a binary with this so we can test directly?

Reply:
@CyrusNajmabadi I can't get how changing the name in the compiler is even a problem for the IDE-side?

If, for example, the compiler changed it from `<Clone>$` to `<Copy>$`, why this would  affect the IDE?

Reply:
because imagine, for the sake of argument that the IDE impl was a hard block on showing teh exact string "<Clone>$".  The the test woul still pass because we woudln't have such an item in teh list.  But the user experience woudl still be bad because `<Copy>$` is now in the list.  

THis check validates that we do not show the compiler 'clone' method in the list, regardless of what they name it.

Reply:
> THis check validates that we do not show the compiler 'clone' method in the list, regardless of what they name it.

The test verifies that `cloneMemberName` is absent from the completion list. If the method was renamed to `<Copy>$`, the test is still verifying `cloneMemberName` which should have the new name. Am I still missing something?

Reply:
yes.  that's why the test is correct.  because it's validating that the name the compiler is using does not show up. meaning that the compiler can change that name and the test will continue passing.  as opposed to if we only hardcoded an exception for `<Clone>$` which would make the test still pass, even if the user scenario was broken.